packages/gatsby-theme-blog/README.md

```diff
@@ -66,7 +66,10 @@ module.exports = {
 | `contentPath`            | `content/posts`  | Location of blog posts                                                                                                                                             |
 | `assetPath`              | `content/assets` | Location of assets                                                                                                                                                 |
 | `mdxOtherwiseConfigured` | `false`          | Set this flag `true` if `gatsby-plugin-mdx` is already configured for your site.                                                                                   |
+| `preset`  | `gatsby-theme-ui-preset`          | Install your dependency and pass in the package name. All local shadowing will deep merge with this preset. |
```

Comment:
Not sure how to make this more clear, but the language for these options sounds a little odd to me -- could this be a noun phrase to describe what the value should be instead of an imperative sentence? And maybe the merging behavior with shadowed files can be explained below?

Reply:
Full README changes are in this PR: https://github.com/gatsbyjs/themes/pull/12
I'll also update this language.